### PR TITLE
Bug 1212402 - [TV 2.5] Apply visual spec to the TV remote control app

### PR DESCRIPTION
**Pull request by luke-chang**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4cecff13d4b99cbcbf99fe57d2452c1442e915b6

File: tv_apps/remote-control/js/section.js [hunk 1]
```diff
@@ -68,6 +68,14 @@
       if (activeChildSection) {
         activeChildSection.handleMove(direction);
       } else {
+        if (this._handleMove) {
```
Comment (rexboy7) [inline tv_apps/remote-control/js/section.js]: I didn't see this being initialized anywhere.

Comment (rexboy7) [inline tv_apps/remote-control/js/section.js]: Oh, so this is for `SettingsSection`? I wonder if we have better way to write this kind of "polymorphism"..

File: tv_apps/remote-control/js/settings_section.js [hunk 2]
```diff
@@ -28,6 +28,15 @@
...
   // Section's Interface
+  SettingsSection.prototype._handleMove = function(direction) {
+    var elem = this.getFocusedElement();
+    if ((elem.id == 'option-pair-on' && direction == 'down') ||
+        (elem.id == 'option-disabled' && direction == 'up')) {
+      return document.getElementById('clear-authorized-devices');
+    }
+  };
+
+  // Section's Interface
```
Comment (rexboy7) [inline tv_apps/remote-control/js/settings_section.js]: Repeated?

File: tv_apps/remote-control/js/section.js (hunk 1 above)
Comment (rexboy7) [inline tv_apps/remote-control/js/section.js]: How about adding an empty `Section.prototype._spatialNav()`?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=932a4d3159c0d367761b78e41087818bf2b92670

**Pull request merged**